src/groupeddataframe/splitapplycombine.jl

```diff
@@ -460,9 +495,10 @@ function do_call(f::Any, idx::AbstractVector{<:Integer},
 end
 function do_call(f::Any, idx::AbstractVector{<:Integer},
                  starts::AbstractVector{<:Integer}, ends::AbstractVector{<:Integer},
-                 gd::GroupedDataFrame, incols::NamedTuple, i::Integer)
+                 gd::GroupedDataFrame, incols::Tuple, i::Integer)
+    # we retain indcols as NamedTuple here to allow in the future not to auto-splat
```

Comment:
This appears to go against what the code does?

Reply:
removed - it was leftover from an initial design, when I left `NamedTuple`